internal/v1/server.go

```diff
@@ -480,9 +480,11 @@ func (h *Handlers) ComposeImage(ctx echo.Context) error {
 
 	err = h.server.db.InsertCompose(composeResult.Id, idHeader.Identity.AccountNumber, idHeader.Identity.Internal.OrgId, rawCR)
 	if err != nil {
+		h.server.logger.Info("Error inserting id into db", err)
```

Comment:
this can be a `.Error` level alert, useful when filtering messages by level in kibana